src/axom/multimat/multimat.hpp

```diff
@@ -216,6 +208,29 @@ class MultiMat
    */
   void setNumberOfCells(int num_cells);
 
+  /**
+   * \brief Sets the allocator ID to use for allocations.
+   *
```

Comment:
Looking at the implementation, this function calls `setSlamAllocatorID(alloc_id)` and `setFieldAllocatorID(alloc_id)`.
Should we mention this in the doxycomment?